lib/discourse_code_review/github_repo.rb

```diff
@@ -162,10 +162,16 @@ def commits_since(ref = nil, merge_github_info: true, pull: true, single: false)
     end
 
     def followees(ref)
-      git_repo
-        .trailers(ref)
-        .select { |x| x.first == 'Follow-up-to' }
-        .map(&:second)
+      result = []
+
+      git_repo.commit(ref).message.lines.each do |line|
+        data = line.match(/follow.*?([a-z0-9]{7,})/i)
+        if data.present?
+          result << data[1]
+        end
```

Comment:
I think a hash is only hexadecimal characters?

You can also shorten that code a bit if you want.

```suggestion
        data = line[/follow.*?(\h{7,})/i, 1]
        result << data if data
```